docs/guides/guards.md

```diff
@@ -247,6 +250,13 @@ const doorService = interpret(doorMachine)
   .start();
 // => { closed: 'idle' }
 
+doorService.send('OPEN');
+// => { closed: 'idle' }
+
+doorService.send('SET_ALARM');
+// => { closed: 'idle' }
```

Comment:
it would be more understandable for me if those 2 lines of comments would be reordered

Reply:
however - i think the other fix for this (updating the initial context value) would be good enough

Reply:
I think it's good as-is because we want to demonstrate that the door will not go to `{ closed: 'error' }` if `alert: false`.